webClient/package.json

```diff
@@ -36,31 +35,33 @@
     "@angular/platform-browser-dynamic": "~6.0.9",
     "@angular/router": "~6.0.9",
     "@types/node": "^6.0.112",
+    "@zlux/widgets": "git+https://github.com:zowe/zlux-widgets.git",
     "@zowe/zlux-angular-file-tree": "0.0.3-staging-6-20200521223819",
     "angular2-template-loader": "~0.6.2",
     "codelyzer": "~4.4.2",
-    "compression-webpack-plugin": "~2.0.0",
-    "copy-webpack-plugin": "~4.5.2",
+    "compression-webpack-plugin": "~3.1.0",
+    "copy-webpack-plugin": "~5.1.1",
     "core-js": "~2.5.7",
     "css-loader": "~1.0.0",
     "exports-loader": "~0.7.0",
     "file-loader": "~1.1.11",
     "html-loader": "~0.5.5",
-    "node-sass": "~4.11.0",
+    "monaco-editor": "^0.20.0",
+    "monaco-editor-webpack-plugin": "^1.9.0",
     "rxjs": "~6.2.2",
     "rxjs-compat": "~6.2.2",
-    "sass-loader": "~7.1.0",
+    "sass": "~1.26.0",
+    "sass-loader": "~8.0.0",
     "source-map-loader": "~0.2.4",
     "style-loader": "~0.22.1",
     "ts-loader": "~4.4.2",
     "ts-node": "~4.1.0",
     "tslint": "~5.10.0",
-    "typescript": "~2.7.2",
+    "typescript": "~3.7.0",
```

Comment:
Angular 9 supports 3.7 I believe, Angular 8 supports 3.4

So don't remember what Angular version we're responsible for (6?), just making sure you have the version you intended it to be in this PR

Reply:
Yeah, it's not because I wanted to but because monaco itself needs typescript of a certain version. This seems to work even though angular doesnt officially support it. I think it just means "dont do newer typescript features with older angular", which is fine because here, the newer typescript features are being used for monaco, which lives outside of angular.